api/src/shared/game-time.ts

```diff
@@ -0,0 +1,67 @@
+import { UniversePropertyRepo } from '../db/index';
+
+const GameTimeKey = 'universal.last-saved-game-time';
```

Comment:
Probably those const should be part of UniversePropertyRepo or UniverseTimeEntity

Reply:
role UniversePropertyRepo and entity is to provided way to store any general universe property. why it should know how exactly it used?
IMO like this is better - there is one direction relation. `game-time` module use `universe-property-repo` module, but `universe-property-repo` know nothing about who and how use it.